index.js

```diff
@@ -55,3 +62,25 @@ const endWhenStreamsDone = async (passThroughStream, streams) => {
 		}
 	}
 };
+
+// Ideally we would re-use the same `AbortController` for all input streams.
+// However, this creates a `maxListeners` warning when there are more than 10 input streams.
+// This cannot be fixed because `AbortController.signal` is an `EventTarget`, which does not have a `.setMaxListeners()` method.
```

Comment:
Source code in Node.js:

https://github.com/nodejs/node/blob/0f461aad4cbf747fa053714f08e5030ddf019977/lib/internal/abort_controller.js#L356C22-L356C33

https://github.com/nodejs/node/blob/0f461aad4cbf747fa053714f08e5030ddf019977/lib/internal/event_target.js#L562-L577

We could technically set `abortController.signal.kMaxEventTargetListeners = number` because that property is exposed, but it is undocumented.